src/voprf.rs

```diff
@@ -600,7 +600,23 @@ impl<G: Group, H: BlockInput + Digest> BlindedElement<G, H> {
     #[cfg(test)]
     /// Only used for testing zeroize
     pub fn as_ptrs(&self) -> Vec<Vec<u8>> {
-        vec![self.value.to_arr().to_vec()]
+        alloc::vec![self.value.to_arr().to_vec()]
+    }
+
+    #[cfg(feature = "internal")]
+    /// Unsafely set the value. This should be used with caution, since
+    /// it does not perform any checks on the validity of the value itself!
+    pub fn set_value_unsafe(value: G) -> Self {
+        Self {
+            value,
+            hash: PhantomData,
+        }
+    }
```

Comment:
"unsafe" has a specific meaning in Rust, which doesn't apply here, I would replace it with "dangerous". Also `set_value` may not be appropriate, it implies an instantiation already exists, maybe `dangerous_from_group` might be better?

Reply:
Good suggestion, I renamed to `from_value_unchecked`